Sources/JWTKit/RSA/RSA.swift

```diff
@@ -0,0 +1,289 @@
+@preconcurrency import _CryptoExtras
+import Crypto
+import Foundation
+import SwiftASN1
+import X509
+
+/// Namespace encompassing functionality related to the RSA (Rivest–Shamir–Adleman) cryptographic algorithm.
+/// Relatively to other algorithms such as ECDSA and EdDSA, RSA is considered slow and should be avoided when possible.
+public enum RSA: Sendable {}
```

Comment:
I'd be tempted to shove this under an `Unsafe` namespace similar to Swift Crypto - thoughts @gwynne?

Reply:
Strongly concur.